src/website/views/components/modal.pug

```diff
@@ -43,6 +43,78 @@ h4 Base
       </div>
     </div>
 
+h4 Scrollable
+p.-text
+  | If the content height is bigger than the content modal container
```

Comment:
Change to:
Enable scrolling if the height of the modal's content is larger than the modal's content container.

Reply:
done